src/library/Hooks/useUrlVars/index.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright 2022 @paritytech/polkadot-staking-dashboard authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { NETWORKS } from 'config/networks';
+import { useApi } from 'contexts/Api';
+import { availableLanguages } from 'locale';
+import { changeLanguage } from 'locale/utils';
+import { useTranslation } from 'react-i18next';
+import { NetworkName } from 'types';
+import { extractUrlValue } from 'Utils';
+
+export const useUrlVars = () => {
+  const { i18n } = useTranslation();
+  // get url variables
+  const networkFromUrl = extractUrlValue('n');
+  const lngFromUrl = extractUrlValue('l');
```

Comment:
These can be inside `initialise()` at the top. They are a part of `initialise`- although there is no harm in calling them here (app will still work) let's keep all initialisation logic together.